src/include/network/lan/address.rb

```diff
@@ -1174,7 +1174,7 @@ def initIfplugdPriority(_key)
       UI.ChangeWidget(
         Id("IFPLUGD_PRIORITY"),
         :Value,
-        Builtins.tointeger(Ops.get_string(@settings, "IFPLUGD_PRIORITY", "0"))
+        @settings["IFPLUGD_PRIORITY"].to_i
```

Comment:
I don't see the difference. Maybe we should update the docs of tointeger:
http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast/Builtins#tointeger-class_method
http://www.rubydoc.info/stdlib/core/String#to_i-instance_method